src/components/ErrorBanner.js

```
@@ -0,0 +1,41 @@
+    this.banner.innerHTML = '';
+    errs.forEach((err) => {
+      const span = document.createElement('span');
+      span.textContent = err;
+      span.style.marginRight = '1em';
+      this.banner.appendChild(span);
+    });
+    this.banner.style.display = '';
+  }
+
+  destroy() {
+    this.unsubscribe && this.unsubscribe();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)